crates/red_knot_python_semantic/resources/mdtest/type_properties/is_assignable_to.md

```diff
@@ -331,6 +331,25 @@ static_assert(is_assignable_to(Intersection[LiteralString, Not[Literal[""]]], No
 static_assert(is_assignable_to(Intersection[LiteralString, Not[Literal["", "a"]]], Not[AlwaysFalsy]))
 ```
 
+## Generics
+
+### Inheriting from a specialized generic class
+
+```py
+from knot_extensions import is_assignable_to, static_assert
+from typing import Any
+
+class A[T]:
+    pass
+
+class B(A[int]):
+    pass
+
+static_assert(is_assignable_to(B, A[int]))
+static_assert(not is_assignable_to(B, A[bool]))
+static_assert(not is_assignable_to(B, A[Any]))
```

Comment:
Should this one be assignable?

Reply:
Yes, it should be, due to the rules of materialization: that `Any` can materialize to `int`.

Reply:
Done (in new tests)